build.gradle

```diff
@@ -14,16 +14,16 @@ repositories {
 
 dependencies {
   implementation "io.gatling:gatling-enterprise-plugin-commons:1.0.0"
-  implementation "com.github.jengelman.gradle.plugins:shadow:5.2.0"
-  implementation "org.apache.logging.log4j:log4j-core:2.17.1"
+  implementation "org.apache.ant:ant:1.10.11"
```

Comment:
For `org.apache.tools.zip`; same as was used with `ShadowJar`. Also the same used by Gradle's native `Jar`/`Zip` tasks, but Gradle's internal version of the lib gets shaded to an `org.gradle.internal.impldep.*` package, so we presumably shouldn't use it directly.